pandas/tests/series/indexing/test_setitem.py

```diff
@@ -135,6 +136,17 @@ def test_setitem_boolean_td64_values_cast_na(self, value):
         expected = Series([NaT, 1, 2], dtype="timedelta64[ns]")
         tm.assert_series_equal(series, expected)
 
+    def test_setitem_boolean_nullable_int_types(self, any_nullable_int_dtype):
+        # GH: 26468
+        ser = Series(pd_array([5, 6, 7, 8], dtype=any_nullable_int_dtype))
```

Comment:
No need for the `pd.array` here, since the Series constructor will handle the nullable dtype as well.

Reply:
Thx, removed it